### ISSUE TITLE
PointOfInterestTemplate - Not working

### ISSUE THREAD

**Issue by ElangoPrince** [birkir/react-native-carplay#159]
**Describe the bug**
How to use **PointOfInterestTemplate** in react native. I have installed and used basic point-of-interest template, but app getting crash. 
**Configuration Details**: 
"react-native": "0.72.6",
"react-native-carplay": "^2.3.0",
"IOS-16",
carplay and map entitlements added. 

**To Reproduce**
"react-native-run-ios"

**Expected behavior**

**Screenshots/Videos**

**CarPlay (please complete the following information):**
 - Device: [e.g. iPhone 13 / Simulator]
 - OS version [e.g. iOS 16]
 - RNCarPlay version [e.g. 2.4.0]

**Android Auto (please complete the following information):**
 - Device: [e.g. Pixel 8 / emulator]
 - Android Version: [e.g. iOS8.1]
 - RNCarPlay version [e.g. 2.4.0]

**Additional context**

**Comment by birkir** [comment]
maps entitlement does not have access to POI, refer to the [entitlement matrix](https://github.com/birkir/react-native-carplay/blob/master/CarPlay.md#entitlement-matrix)

**Comment by saikiranm33** [comment]
@ElangoPrince  Are u able to run the POI ?

**Comment by ElangoPrince** [comment]
@saikiranm33 Nope. Map's entitlement also required to implement POI (Not enough com.apple.developer.carplay-fueling) entitlement. Any how we need both entitlement to implement this feature.
